lib/api.js

```diff
@@ -2133,4 +2133,25 @@ API.prototype.pushNotificationsUnsubscribe = function(cb) {
   });
 };
 
+/**
+ * Returns result object.
+ * @param {String} opts
+ * @param {Int} opts.feePerKb - Fee value
+ * @param {String} opts.excludeUnconfirmedUtxos - Indicates it if should use (or not) the unconfirmed utxos
+ * @param {String} opts.returnInputs - Indicates it if should return (or not) the inputs
+ * @return {Callback} cb - Return error (if exists) and object result
+ */
+API.prototype.getSendMaxInfo = function(opts, cb) {
+  var self = this;
+  var url = '/v1/sendmaxinfo/';
+  if (opts.feePerKb) url += '?feePerKb=' + opts.feePerKb;
```

Comment:
This will not work if `feePerKb` is not specified. Please see how it is done here: https://github.com/JDonadio/bitcore-wallet-client/blob/feat/send-max/lib/api.js#L1591